README.md

```diff
@@ -0,0 +1,84 @@
+# Ada Pets Node
+
+<!-- Note: this was based on Ada Pets React no the other way around. -->
+
+## At a Glance
+
+- Individual, [stage 1](https://github.com/Ada-Developers-Academy/pedagogy/blob/master/rule-of-three.md#stage-1) project.
+- To be completed in class on **DATE**. No pull request is required.
+
+## Learning Goals
+
+* Reading and understanding existing JavaScript code
+* Using axios to read data from an API in JavaScript
+* Parsing through an API response in JavaScript and extracting desired information
+* Using axios to write data to an API in JavaScript
+
+## Introduction
+
+Ada Lovelace is starting a pet rescue service advertising pets in need of rescue via an API.   This app should load the list of pets automatically and present a menu with options.   The user should be able to list all pets, select a pet and see details, add new pets and remove a pet from the App once adopted. 
```

Comment:
Is "app" an appropriate word to use here? (Genuine question, I'm unsure)

Could we mention that this will be a CLI here in the intro? Possibly: "... in need of rescue via an API. In this project, we will build a CLI app for a prospective adoptee .... This app should load the"

Reply:
Oooh this isn't a CLI project -- Whoops!

It'd be awesome if there was language here that reflected that the goal is to implement a series of functions.

One suggestion I have is to make a separate Objective header, much like in [the original js-adagrams README, even if the objective is 2 sentences](https://github.com/AdaGold/adagrams#objective)

Or something else!

Reply:
Okay, final:

Language that clarifies: there is a CLI bit, it's already implemented. The work to focus on here is to implement the functions and make the tests pass